src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Region.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity
+
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.validation.constraints.NotNull
+
+@Entity
+data class Region(
```

Comment:
can we call this `NPSRegion` - it more closely aligns with the terminology i've seen in the DF documentation

Reply:
sure